core/src/tracker.ts

```diff
@@ -1,69 +1,91 @@
 import type { Notifier } from './types';
-import getMaker from './maker';
+import { ethers } from 'ethers';
 import { v4 as uuidv4 } from 'uuid';
+import truncateText from './helpers/truncateText';
 
-const trackTransaction = async function (transactionPromise: Promise<any>, notifier?: Notifier): Promise<string> {
-    const maker = await getMaker();
+const DEFAULT_NOTIFICATION_DURATION = 3;
+const NUMBER_OF_BLOCKS_TO_CONFIRM = 5;
```

Comment:
as above - I'm a bit surprised to see that all of these values are hard-coded. I appreciate that the context is a library rather than an app, but still we should offer ways to customize behavior, right?

Another point on this I raised earlier: With environment variables scattered around all modules it becomes very hard to see what can be configured in the application - how about using the `privateRuntimeConfig` that nuxt offers us, see https://nuxtjs.org/docs/configuration-glossary/configuration-runtime-config/#privateruntimeconfig to define these options in a centralized place?

Reply:
> I'm a bit surprised to see that all of these values are hard-coded

I think that was a standard for us to set those variables at the top of the file as a constant. In case we actually need to configure it, env variables or $config wouldn't be the best approach, since `/core` is not part of the nuxt. I outlined the approach in a new issue https://github.com/sidestream-tech/unified-auctions-ui/issues/59, please comment there if you think this approach is viable. And let's address it in a separate PR!

Reply:
Sounds good to me :+1: